x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elasticsearch/CustomElandInternalServiceSettings.java

```diff
@@ -21,7 +19,7 @@
 
 import static org.elasticsearch.TransportVersions.ML_TEXT_EMBEDDING_INFERENCE_SERVICE_ADDED;
 
-public class CustomElandInternalServiceSettings extends TextEmbeddingInternalServiceSettings {
+public class CustomElandInternalServiceSettings extends ElasticsearchInternalServiceSettings {
```

Comment:
Perhaps this class should be renamed to `CustomElandElasticsearchServiceSettings`?